tendrl/commons/objects/cluster/atoms/create/__init__.py

```diff
@@ -5,9 +5,9 @@
 from tendrl.commons.flows.create_cluster.gluster_help import create_gluster
```

Comment:
we no longer support create cluster, please remove this file and supporting files

Reply:
i will remove it